Project/dotnet-core/ConsoleApp/Program.cs

```diff
@@ -0,0 +1,217 @@
+﻿/*  Copyright (c) MediaArea.net SARL. All Rights Reserved.
+ *
+ *  Use of this source code is governed by a BSD-style license that can
+ *  be found in the License.html file in the root of the source tree.
+ */
+
+//+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+//
+// Microsoft Visual C# example for .NET Core console apps
+//
+// To make this example work, you must put MediaInfo.Dll
+// in the "./bin/__ConfigurationName__/netcoreapp3.1/" folder with the other binaries
+// and add MediaInfoLib/Source/MediaInfoDLL/MediaInfoDLL.cs to your project
+//
+//+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using System.Runtime.InteropServices;
+using MediaInfoLib;
+
+using Azure;
+using Azure.Storage.Blobs;
+using Azure.Storage.Blobs.Specialized;
+
+namespace ConsoleApp
+{
+    class Program
+    {
+        static async Task Main(string[] args)
+        {
+            String ToDisplay;
+            MediaInfo MI = new MediaInfo();
+
+            ToDisplay = MI.Option("Info_Version", "0.7.0.0;MediaInfoDLL_Example_CS;0.7.0.0");
+            if (ToDisplay.Length == 0)
+            {
+                Console.WriteLine("MediaInfo.Dll: this version of the DLL is not compatible");
+                return;
+            }
+
+            if (ToDisplay == "Unable to load MediaInfo library")
```

Comment:
I would have like to use more explicit .LoadLibrary() / .IsLoaded() here, but wanted to limit my changes to only demonstrating the Azure byte-range pattern.